core/test/run_models_test.jl

```diff
@@ -395,8 +395,8 @@ end
     @test only(model.integrator.sol(0day)) == 1000.0
     # constant UserDemand withdraws to 0.9m/900m3
     @test only(model.integrator.sol(150day)) ≈ 900 atol = 5
-    # dynamic UserDemand withdraws to 0.5m/509m3
-    @test only(model.integrator.sol(180day)) ≈ 509 atol = 1
+    # dynamic UserDemand withdraws to 0.5m/503m3
```

Comment:
This is still too coarse a test. You should have a look at `DataFrame(Ribasim.flow_table(model))`, and check whether the outflow to inflow proportion matches your transient return factor

Reply:
What is the relation between out-to-inflow proportion and this test?

Reply:
Timepoint at 179day show a return ratio of 0.197ish and return factor is 0.2